### PR TITLE
FUSETOOLS-2215 - do not generate ID attributes for unsupported types

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java [hunk 1]
```diff
@@ -142,16 +140,38 @@ private String computeId(Node child) {
...
+	
+	private boolean ignoreNode(Node child) {
+		return !isCamelNamespaceElement(child) && !isSupportedGlobalType(child);
+	}
+	
+	private boolean isCamelNamespaceElement(Node child) {
+		return startsWithNamespace(child, "http://camel.apache.org/schema/");
+	}
+	
+	private boolean isBlueprintNamespaceElement(Node child) {
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java]: finally, this method is not used anymore with the new implementation, isn't it?

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java [hunk 2]
```diff
@@ -142,16 +140,38 @@ private String computeId(Node child) {
...
+		return startsWithNamespace(child, "http://www.osgi.org/xmlns/blueprint/"/*v1.0.0*/);
+	}
 
+	private boolean isSpringNamespaceElement(Node child) {
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java]: finally, this method is not used anymore with the new implementation, isn't it?

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java (hunk 1 above)
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java]: Not used atm. Not sure if its worth cleaning them up atm.

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java (hunk 2 above)
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/CamelFile.java]: Not used atm. Not sure if its worth cleaning them up atm.

**Comment by lhein** [comment]
testPR

**Pull request merged**